doc/nrf/ug_multi_image.rst

```diff
@@ -129,58 +132,88 @@ For example:
 Variables in the parent image are not propagated to the child image, with the following exceptions:
 
-* Any variable named *IMAGENAME*\_FOO in a parent image is propagated to the child image named *IMAGENAME* as FOO.
-* Variables that are in the list ``SHARED_MULTI_IMAGE_VARIABLES`` are propagated to all child images.
+* Any variable named ``<IMAGE_NAME>\_FOO`` in a parent image is propagated to the child image named ``<IMAGE_NAME>`` as ``FOO``.
+* Variables that are in the ``SHARED_MULTI_IMAGE_VARIABLES`` list are propagated to all child images.
 
 With these two mechanisms, it is possible to set variables in child images from either parent images or the command line, and it is possible to set variables globally across all images.
-For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows::
+For example, to change the ``CONF_FILE`` variable for the MCUboot image and the parent image, specify the CMake command as follows:
+
+.. code-block::
 
    cmake -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
 
 You can extend the CMake command that is used to create the child images by adding flags to the CMake variable ``EXTRA_MULTI_IMAGE_CMAKE_ARGS``.
 For example, add ``--trace-expand`` to that variable to output more debug information.
 
+With ``west``, these configuration variables are passed into CMake by using the ``--`` separator:
+
+.. code-block::
+
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -Dmcuboot_CONF_FILE=prj_a.conf -DCONF_FILE=app_prj.conf
+
+Any configuration option for the application or child image can be passed in this way.
+For example, use the following command to build the ``hello_world`` sample using multi-stage bootloaders and configurations specific to each bootloader image:
+
+.. code-block::
+
+   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- -DCONFIG_SECURE_BOOT=y -DCONFIG_BOOTLOADER_MCUBOOT=y -DCONFIG_BUILD_S1_VARIANT=y -Db0_OVERLAY_CONFIG=overlay-minimal-size.conf -Dmcuboot_CONFIG_LOG_MINIMAL=y
+
+The options given in the command do the following:
+
+* Enable the setting for the first-stage immutable ``b0`` bootloader - :option:`CONFIG_SECURE_BOOT`.
+* Enable the setting for the second-stage upgradable MCUBoot - :option:`CONFIG_BOOTLOADER_MCUBOOT`.
+* Produce a pre-signed variant of the MCUBoot image for ``S1`` - :option:`CONFIG_BUILD_S1_VARIANT`.
+* Build the ``b0`` child image with one of its own overlays.
+* Build the MCUBoot child image with minimal logging features - :option:`CONFIG_LOG_MINIMAL`.
+
+Configurations passed using this method are called :ref:`one-time cmake arguments <west-building-cmake-args>`.
+To always build the application with these configurations, place them into an overlay, the base ``prj.conf`` file, or a custom ``prj.conf`` file also pointed to using a one-time argument, as demonstrated above.
```

Comment:
Overlays are used by dts.
This is a Kconfig fragment.

Also, the `prj.conf` is just the samples `prj.conf`, and a custom conf file can be name anything.
Doesn't have to be named `prj.conf`.

> To always build the application with these configurations

Which configurations ?
Placing `mcuboot_CONFIG_LOG_MINIMAL=y` in my applications `prj.conf` is not going to work.
That setting would have to go into the config file for `mcuboot`.
That is not clear from this text.

Reply:
Addressed in [this commit](https://github.com/nrfconnect/sdk-nrf/pull/3607/commits/8f78f39ea2d00a43e1149293706416747f825acd) for #3607.